Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -249,18 +283,44 @@ define([
             collectionOptions.basePath = this._url;
         }
 
-        var ellipsoid = Ellipsoid.WGS84;
-        var position = new Cartesian3();
         var instances = collectionOptions.instances;
         byteOffset = 0;
 
+        var translation = new Cartesian3();
+        var normalUp = new Cartesian3();
+        var normalRight = new Cartesian3();
+        var normalOut = new Cartesian3();
+        var rotation = new Matrix3();
         for (var i = 0; i < instancesLength; ++i) {
-            // Get longitude and latitude
-            var longitude = instancesView.getFloat64(byteOffset, true);
-            byteOffset += sizeOfFloat64;
-            var latitude = instancesView.getFloat64(byteOffset, true);
-            byteOffset += sizeOfFloat64;
-            var height = 0.0;
+            // Get and decode x, y, z
+            translation.x = instancesView.getUint16(byteOffset, true) * scaleX + translateX;
+            byteOffset += sizeOfUint16;
+            translation.y = instancesView.getUint16(byteOffset, true) * scaleY + translateY;
+            byteOffset += sizeOfUint16;
+            translation.z = instancesView.getUint16(byteOffset, true) * scaleZ + translateZ;
+            byteOffset += sizeOfUint16;
+
+            // Get encoded orientation vectors
+            var normalOneX = instancesView.getUint8(byteOffset);
+            byteOffset ++;
+            var normalOneY = instancesView.getUint8(byteOffset);
+            byteOffset ++;
+            var normalTwoX = instancesView.getUint8(byteOffset);
+            byteOffset ++;
+            var normalTwoY = instancesView.getUint8(byteOffset);
+            byteOffset ++;
+
+            // Decode compressed normals
+            AttributeCompression.octDecode(normalOneX, normalOneY, normalUp);
+            AttributeCompression.octDecode(normalTwoX, normalTwoY, normalRight);
+
+            // Compute third normal
+            Cartesian3.cross(normalRight, normalUp, normalOut);
```

Comment:
"Out" here is confusing since, if I am following correctly, we would usually call it "Up" here, e.g., east/north/up; however, "Up" is already used, and perhaps rightfully so.

I don't have a better idea now; will let you know if I come up with something.